plugin/trino-lance/src/test/java/io/trino/plugin/lance/LanceQueryRunner.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.lance;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.trino.metadata.QualifiedObjectName;
+import io.trino.plugin.base.util.Closables;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingTrinoClient;
+import io.trino.tpch.TpchTable;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static io.airlift.units.Duration.nanosSince;
+import static io.trino.plugin.lance.catalog.BaseTable.LANCE_SUFFIX;
+import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+import static java.lang.String.format;
+import static java.nio.file.Files.createTempDirectory;
+import static java.util.Locale.ENGLISH;
+import static java.util.Objects.requireNonNull;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class LanceQueryRunner
+{
+    public static final String LANCE_CATALOG = "lance";
+
+    private LanceQueryRunner() {}
+
+    private static final Logger log = Logger.get(LanceQueryRunner.class);
+
+    public static Builder builder(String warehousePath)
+    {
+        return new Builder(warehousePath);
+    }
+
+    public static void main(String[] args)
```

Comment:
Thanks for adding a query runner main method! This is very helpful for reviews. 

This requires additional JVM options, right? Please leave a code comment if so.